anyway/views/schools/api.py

```diff
@@ -94,8 +94,8 @@ def schools_yishuvs_api():
             ),
             not_(
                 and_(
-                    SchoolWithDescription2020.latitude == None,
-                    SchoolWithDescription2020.longitude == None,
+                    SchoolWithDescription2020.latitude is None,
+                    SchoolWithDescription2020.longitude is None,
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969